core-plugins/src/main/java/co/cask/hydrator/plugin/batch/joiner/Joiner.java

```diff
@@ -52,6 +58,11 @@
   "records from non-required inputs will only be present if they match join criteria. If there are no required " +
   "inputs, outer join will be performed")
 public class Joiner extends BatchJoiner<StructuredRecord, StructuredRecord, StructuredRecord> {
+
+  public static final String JOIN_OPERATION_DESCRIPTIOPN = "Join Operation";
```

Comment:
typo: DESCRIPTIOPN

Reply:
'Join Operation' doesn't really give much information. I think something like 'Used as a key in a join' is more descriptive.

Reply:
Also, I think it would be better to change the tense of the verbs used in these descriptions? Elsewhere, we use verbs like 'writing to' and 'reading from', but I think it makes more sense for them to be 'wrote to' and 'read from'. When you're looking at a lineage diagram, these are all operations that happened in the past.

Reply:
fixed. using past tense for descriptions.